### PR TITLE
#160451549 Setup Redux

### PR DESCRIPTION
**Pull request by Mrshinezee**
#### What does this PR do?
 - setup redux

#### Description of Task to be completed?
- setup redux folder structures

#### How should this be manually tested?
- start the app by running ```yarn start:dev```
the app should open on port ```8080``` on your browser

#### What are the relevant pivotal tracker stories?
[#160451548](https://www.pivotaltracker.com/story/show/160451549, "Setup Redux")

#### Screenshots (if appropriate)
screen shot 2018-10-31 at 7 48 17 am

### CODE REVIEW

File: src/action/sample.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Moduler Importations
+import * as types from "../actionTypes/sample";
```
Comment (KaiserPhemi) [inline src/action/sample.js]: #### Critical
Avoid wildcard imports as much as possible. You can check [here](https://github.com/airbnb/javascript#modules) to get more context

File: src/action/sample.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+export const setPlayerScore = score => {
```
Comment (KaiserPhemi) [inline src/action/sample.js:8]: #### Critical
Always document your functions

File: src/action/sample.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    return {
+      type: types.SAMPLE_LOGIN_SUCCESS,
+      score: score
+    };
+  };
+
```
Comment (KaiserPhemi) [inline src/action/sample.js]: #### Critical
Always append a new line at the end of a file

File: src/reducers/sample.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// modular importation
+import initialState from '../store/initialState';
+import * as types from "../actionTypes/sample";
```
Comment (KaiserPhemi) [inline src/reducers/sample.js]: #### Critical
Same as above

File: src/reducers/sample.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+        return {
+          ...state,
+          ...action.payload
+        };
+      default:
+        return state;
+    }
+  };
+  export default reducer;
+
```
Comment (KaiserPhemi) [inline src/reducers/sample.js]: #### Critical
New line and every other place where this is needed.

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
@Mrshinezee 

#### Critical
Do attend to all feedback.

File: src/action/sample.js (hunk 1 above)
Comment (Mrshinezee) [reply src/action/sample.js]: Alright, I will implement the feedback

**Review by obulaworld — APPROVED** [review]
Looking Good To Me.

**Review by e0works — APPROVED** [review]
Nice work, it looks good to me.

File: src/action/sample.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+ * @param {object} score 
+ * @desc sample of an action
+*/
+export const setPlayerScore = score => {
+    return {
+      type: SAMPLE_LOGIN_SUCCESS,
+      score: score
+    };
+  };
+
```
Comment (KaiserPhemi) [inline src/action/sample.js:14]: #### Critical
New line here???

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
@Mrshinezee

File: src/action/sample.js (hunk 6 above)
Comment (Mrshinezee) [reply src/action/sample.js:14]: @KaiserPhemi please I did not get the feedback. Line 14 is already a new line

Comment (KaiserPhemi) [reply src/action/sample.js:14]: There should be a new line at the end of the file.

Comment (Mrshinezee) [reply src/action/sample.js:14]: screen shot 2018-10-31 at 1 35 41 pm

Comment (Mrshinezee) [reply src/action/sample.js:14]: Line 14 is the new line at the end of the file

Comment (KaiserPhemi) [reply src/action/sample.js:14]: Have you pushed this cos github still flags it?
screen shot 2018-10-31 at 1 38 18 pm

Comment (Mrshinezee) [reply src/action/sample.js:14]: Yes the screenshot is from the PR online

**Review by KaiserPhemi — APPROVED** [review]

**Comment by KaiserPhemi** [comment]
@Mrshinezee 
Please rebase and fix merge conflicts.

**Pull request merged**